### PR TITLE
updated submodule to v0.13.13, updated to eos v0.3.1 ( js procedures)

### PR DESCRIPTION
**Pull request by islandbitcoin**
I'm getting an error when converting the YAML to JSON. 

```
error: TS2322 [ERROR]: Type '"object"' is not assignable to type '"union"'.
        "subtype": "object",
        ~~~~~~~~~
    at file:///home/dread/Start9/thunderhub-wrapper/scripts/services/getConfig.ts:33:9

```
attached screenshot of where i'm getting hung up
Screenshot from 2022-06-26 01-48-40
k

### CODE REVIEW

File: thunderhub.patch [hunk 1]
```diff
@@ -1,6 +1,6 @@
---- thunderhub/arm64v8.Dockerfile	2022-04-20 12:52:38.319638554 -0600
+--- thunderhub/Dockerfile	2022-04-20 12:52:38.319638554 -0600
```
Comment (ProofOfKeags) [inline thunderhub.patch:1]: I'm mildly surprised this patch file works. Did you generate it via a diff?

File: Makefile [hunk 2]
```diff
@@ -5,18 +5,28 @@ VERSION := $(shell yq e ".version" manifest.yaml)
...
 install: thunderhub.s9pk
 	embassy-cli package install thunderhub.s9pk
 
-thunderhub.s9pk: manifest.yaml assets/compat/config_spec.yaml assets/compat/config_rules.yaml image.tar instructions.md
+thunderhub.s9pk: manifest.yaml assets/compat/config_spec.yaml assets/compat/config_rules.yaml image.tar instructions.md scripts/embassy.js 
 	embassy-sdk pack
 
 Dockerfile: $(THUNDERHUB_SRC)
-	cp thunderhub/arm64v8.Dockerfile Dockerfile
+	cp thunderhub/Dockerfile Dockerfile
```
Comment (ProofOfKeags) [inline Makefile:24]: umm this may not be an allowed change. Are we sure this is always arm64 compatible

File: thunderhub.patch (hunk 1 above)
Comment (islandbitcoin) [reply thunderhub.patch:1]: nope. I manually edited it after reverse engineering yours.

File: Makefile (hunk 2 above)
Comment (islandbitcoin) [reply Makefile:24]: the new update to thunderhub included multiarch. the new Dockerfile is now compatible with arm64
https://github.com/apotdevin/thunderhub/releases/tag/v0.13.11

Comment (ProofOfKeags) [reply Makefile:24]: kk

File: thunderhub.patch (hunk 1 above)
Comment (ProofOfKeags) [reply thunderhub.patch:1]: if you attempt to do a build that applies this patch, does it behave as expected?

Comment (islandbitcoin) [reply thunderhub.patch:1]: yes, it does. I have a working version of it running on my 0.3.0.3 box now.

Comment (islandbitcoin) [reply thunderhub.patch:1]: the health check timing seems to be off, but that's a different bug. 
Screenshot from 2022-06-27 12-42-42
Screenshot from 2022-06-27 12-42-34

Comment (ProofOfKeags) [reply thunderhub.patch:1]: hmm k. sounds good then.

**Review by ProofOfKeags — APPROVED** [review]

**Comment by cryptodread** [comment]
> I'm getting an error when converting the YAML to JSON.
> 
> ```
> error: TS2322 [ERROR]: Type '"object"' is not assignable to type '"union"'.
>         "subtype": "object",
>         ~~~~~~~~~
>     at file:///home/dread/Start9/thunderhub-wrapper/scripts/services/getConfig.ts:33:9
> ```
> 
> attached screenshot of where i'm getting hung up Screenshot from 2022-06-26 01-48-40 

should I convert this to a draft PR since there is still something to be fixed here?

**Pull request merged**